fluentd/map.jinja

```diff
@@ -30,4 +37,4 @@
         'destination_path': '/etc/init/fluentd.conf',
         'source_path': 'fluentd.upstart',
     }
-} %}
+}, grain='init', merge=salt.pillar.get('fluentd:init_service')) %}
```

Comment:
It wasn't doing any filtering before. Instead I was doing that filtering using explicit if/else in the state file.